CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ This is a changelog for Piwik platform developers. All changes for our HTTP API'
 
 ### New features 
 * The JavaScript Tracker does now track outlinks and downloads if a user opens the context menu if the `enabled` parameter of the `enableLinkTracking()` method is set to `true`. To use this new feature use `tracker.enableLinkTracking(true)` or `_paq.push(['enableLinkTracking', true]);`. This is not industry standard and is vulnerable to false positives since not every user will select "Open in a new tab" when the context menu is shown. Most users will do though and it will lead to more accurate results in most cases.
+* The JavaScript Tracker now contains the 'heart beat' feature which can be used to obtain more accurate visit lengths by periodically sending 'ping' requests to Piwik. To use this feature use `tracker.setHeartBeatTimer();` or `_paq.push(['setHeartBeatTimer']);`. By default, a ping request will be sent every 15 seconds. You can specify a custom time by passing an argument, eg, `tracker.setHeartBeatTimer(10);` or `_paq.push(['setHeartBeatTimer', 10]);`.
 
```

Comment:
should be name it maybe `enableHeartBeatTimer(heartBeatDelay)` instead of `setHeartBeatTimer(heartBeatDelay)`? I find it a bit weird to call by default `setHeartBeatTimer()` which is a `set*` method without parameter and it actually enables it, it doesn't `set` anything (apart from changing the default beat delay).

> By default, a ping request will be sent every 15 seconds.

It is most likely not MVP. It would be maybe nice to increase this value over time automatically? Eg if I have the tab open for one hour I don't think a request every 15 seconds is needed, every 2 minutes or so should do. As said probably not MVP